components/table/table-col-sort-button.js

```diff
@@ -107,11 +117,20 @@ export class TableColSortButton extends FocusMixin(LitElement) {
 		`;
 	}
 
+	static #columnDataTypeKeys = {
+		words: ['components.table.a-to-z', 'components.table.z-to-a'],
+		numbers: ['components.table.low-to-high', 'components.table.high-to-low'],
+		dates: ['components.table.old-to-new', 'components.table.new-to-old']
+	};
+
+	static #buttonDescription = ['components.table.change-sort-order', 'components.table.add-sort-order'];
+
 	constructor() {
 		super();
 		this.nosort = false;
 		this.desc = false;
 		this.hasSibling = false;
+		this.columnDataType = 'words';
```

Comment:
nit: we usually alphabetize these (though I see that hasn't really been followed at this point)